src/scss/_components.skip-link.scss

```diff
@@ -2,7 +2,7 @@
   @include body-copy;
   font-weight: 700;
   display: block;
-  left: 0;
+  right: 100%;
```

Comment:
This change is pushing the skip link off the page when you tab to it
 
![skip-link](https://user-images.githubusercontent.com/23404383/60976779-53a08d80-a2fc-11e9-8a8f-407159546952.jpg)

Reply:
🤦‍♂